File: tourFinder/src/main/java/fmi/java/web/tourFinder/model/TourPicture.java
package fmi.java.web.tourFinder.model;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import org.hibernate.annotations.GenericGenerator;

import javax.persistence.*;

@Entity
@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
@Table(name = "tours_pictures")
public class TourPicture {
    @Id
    @GeneratedValue(generator = "system-uuid")
    @GenericGenerator(name = "system-uuid", strategy = "uuid")
    private String id;
    @Lob
    private String picture;
    private String tourId;
}

File: frontEnd/src/models/TourPicture.ts
export class TourPicture{
  public id: string;
  public picture: string;
  public tourId: string;

  constructor(id: string, picture: string, tourId: string) {
    this.id = id;
    this.picture = picture;
    this.tourId = tourId;
  }
}
